lametro/management/commands/compile_pdfs.py

```diff
@@ -221,14 +176,56 @@ def findEventAgendaPacket(self, all_documents=False):
                     ON i.bill_id=d_bill.bill_id
                     INNER JOIN councilmatic_core_eventdocument as d_event
                     ON i.event_id=d_event.event_id
-                    INNER JOIN councilmatic_core_bill AS b
-                    ON d_bill.bill_id=b.ocd_id
-                    WHERE d_bill.bill_id in ( {} )
-                    ) AS subq
+                ) AS subq
                 GROUP BY event_id, event_agenda
-                '''.format(psql_ready_ids)
+            '''
+
+            event_agendas = self.connection.execute(sa.text(query))
+        else:
+            # The compile script typically runs without an `all_documents` argument.
+            # In those cases, the query should only grab the related event_ids for new bill and event documents.
+            grab_events = '''
+            SELECT DISTINCT event_id
+            FROM councilmatic_core_eventagendaitem as i
+            INNER JOIN new_billdocument as bill_doc
+            ON i.bill_id=bill_doc.bill_id
```

Comment:
I think this is a little easier to read, since I don't have to remember aliases.

```sql
SELECT DISTINCT event_id
FROM councilmatic_core_eventagendaitem
INNER JOIN new_billdocument
USING (bill_id)
...
```
if you don't like that, please use more verbose aliases.

Reply:
I like that a lot.